js/main.js

```diff
@@ -0,0 +1,99 @@
+
+var $view = document.querySelectorAll('.view');
+
+document.addEventListener('click', viewHandler);
+
+function swapView(viewName) {
+  if (!event.target.matches('.select')) {
+    return;
+  }
+  for (var i = 0; i < $view.length; i++) {
+    var $dataView = event.target.getAttribute('data-view');
+    var $newView = $view[i].getAttribute('data-view');
+    if ($dataView === $newView) {
+      $view[i].className = 'container view';
+    } else {
+      $view[i].className = 'container view hidden';
+    }
+  }
+  data.view = viewName;
+}
+
+function viewHandler(event) {
+  if (!event.target.matches('.button')) {
+    return;
+  }
+  swapView(event.target.getAttribute('data-view'));
+}
+
+var $input = document.querySelector('#searchData');
+
+$input.addEventListener('submit', submitData);
+
+var characterSearched;
+
+function submitData(event) {
+  event.preventDefault();
+  characterSearched = {
+    searched: $input.input.value
+  };
```

Comment:
Instead of making this a global variable, you should just pass it as an argument to the `renderChars` function.